docs/book/checkout.rst

```diff
@@ -0,0 +1,259 @@
+.. index::
+   single: Checkout
+
+Checkout
+========
+
+**Checkout** is a process that begins when the Customer decides to finish their shopping and transform their **Cart** into an **Order**.
+
+Checkout State Machine
+----------------------
+
+The Order Checkout state machine has 5 states available: ``cart``, ``addressed``, ``shipping_selected``, ``payment_selected``, ``completed``
+and a set of defined transitions between them - which are saved as the **checkoutState** of the **Order**.
```

Comment:
Because the subject changes in the middle of the sentence is not clear whether **states** or **transitions** are saved in the **checkoutStates** (despite the obvious naming 😄 ). It might be confusing on someone's first contact with Sylius.